docs/public/site.webmanifest

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "",
-  "short_name": "",
+  "name": "Mozilla Social",
```

Comment:
Actually I think this is the wrong file. This webmanifest is for the dev docs

Reply:
oooh good call, lemme find where this should be done...

Reply:
Digging into this more, it looks like Elk auto-generates a manifest file: https://github.com/MozillaSocial/elk/blob/main/config/pwa.ts

The generated file exists on prod: https://mozilla.social/manifest-en-US.webmanifest

So we need to figure out why, despite having the manifest, PWA isn't working on prod even though it is working on staging (according to @marcin-kozinski on Slack; I don't have Android to test this)

Reply:
I've removed the change to the manifest. It looks like the autogenerated one (defined with properties at `/modules/pwa/i18n.ts`) should cover it.

If you approve it we can get this merged in